tools/topology/topology2/include/pipelines/cavs/gain-capture.conf

```diff
@@ -52,13 +52,21 @@ Class.Pipeline."gain-capture" {
 		copier."1" {
 			copier_type	"host"
 			type	"aif_out"
-			num_audio_formats 2
+			num_audio_formats 3
 			# 16-bit 48KHz 2ch
 			Object.Base.audio_format.1 {
 				dma_buffer_size "$[$obs * 2]"
 			}
-			# 32-bit 48KHz 2ch
+			# 24-bit 48KHz 2ch
 			Object.Base.audio_format.2 {
+				in_bit_depth		24
+				in_valid_bit_depth	24
```

Comment:
is this correct? I would have thought in_bit_depth = 32 and in_valid_bit_depth = 24?

Reply:
> is this correct? I would have thought in_bit_depth = 32 and in_valid_bit_depth = 24?

Thanks @plbossart. You are right, The container of S24_LE format is 32 bits.